.circleci/config.yml

```diff
@@ -58,6 +59,7 @@ jobs:
           name: run tests
           command: |
             . venv/bin/activate
+            cd lambdas && pytest --cov=./ && cd ..
```

Comment:
change to `--cov=./**.py` to avoid coverage report including test data (notebooks)